crates/turborepo-lib/src/cli.rs

```diff
@@ -552,7 +553,7 @@ pub async fn run(repo_state: Option<RepoState>) -> Result<Payload> {
 
             match command {
                 Some(command) => daemon::daemon_client(command, &base).await,
-                None => daemon::daemon_server(&base, idle_time).await,
+                None => daemon::daemon_server(&base, idle_time, logger).await,
```

Comment:
I'm wondering if we should put `logger` into `CommandBase`, since we already have a UI object there.